Presa height update

georgebrata opened the pull request.

adonisjoe committed:
fix the menu icon on mobile

menu icon now has a fix width and height of 3rem

adonisjoe committed:
fix long text on testimonial

heading text on testimonial page is truncated to 2 lines. heading description on testimonial page is truncated to 3 lines

adonisjoe committed:
Merge https://github.com/adonisjoe/asn-landing into fix-update-ASN
Merged Successfully

adonisjoe committed:
Merge pull request #1 from georgebrata/main

Update

adonisjoe committed:
Make “Trimite mesaj” and “Suna” buttons the same size on mobile

add fixed width of 12rem and remove dedundant paddings that make buttons unequal in height on mobile display.

adonisjoe committed:
Make “Fix the “Contacteaza” button on mobile

add utility class btn--text for fix.

adonisjoe committed:
center all the item vertically

center all the items vertically on testimonial section.

adonisjoe committed:
update the icons to match the title.

change the exisiting icons with the required ones.

georgebrata commented on script.js:
```diff
@@ -10,13 +10,31 @@ const nav = document.querySelector('.nav');
+const slideContainer = document.querySelector('.slider');
+const spinner = document.querySelector('.lds-ellipsis');
+
+console.log(spinner);
```
Please remove the console.log() or comment it. We don't want console logs in production.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1024 @@
-<!DOCTYPE html>
 <html lang="en">
```
Please update the language to "ro".

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1024 @@
+          id="logo"
+          designer="Joe"
+          data-version-number="3.0"
+          style="opacity: 1"
```
Please refactor inline styles to a dedicated CSS class.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1024 @@
+        />
+        <ul class="nav__links">
+          <li class="nav__item">
+            <a class="nav__link" href="#despre" style="opacity: 1">Despre</a>
```
Please refactor inline styles to a dedicated CSS class.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1024 @@
+            <a
+              class="nav__link nav__link--btn btn--show-modal"
+              href="#contact"
+              style="opacity: 1"
```
Please refactor inline styles to a dedicated CSS class.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1024 @@
+            <a
+              class="nav__link nav__link--btn btn--show-modal"
+              href="#contact"
+              style="opacity: 1"
```
Please refactor inline styles to a dedicated CSS class.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1024 @@
+        <input type="email" />
+        <label for="message">Mesaj</label>
+        <textarea
+          style="resize: none"
```
Please refactor inline styles to a dedicated CSS class.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1024 @@
-    </div>
-  </section>
+      <div class="slider">
+        <div class="slide" style="transform: translateX(-800%)">
```
Please refactor inline styles to a dedicated CSS class. For this and following classes I would create CSS classes like `t-8`, `t-7`, etc and use them to keep the HTML more clean.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1024 @@
+            </address>
+          </div>
+        </div>
+        <div class="slide" style="transform: translateX(-700%)">
```
Please refactor inline styles to a dedicated CSS class. For this and following classes I would create CSS classes like `t-8`, `t-7`, etc and use them to keep the HTML more clean.

adonisjoe replied:
I removed the section because we are rendering dynamically from the server

georgebrata reviewed (commented):
Please implement the fixes I suggested.

adonisjoe committed:
highlighted comments fix

adonisjoe committed:
fix despre section buttons

fix the buttons to fit on small screen sizes(Reduced the min-width)

adonisjoe committed:
fix Ai nevoie text

the horizontal padding is reduced to enable the text fit in one line on mobile display(screenshot attached.)

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
-<!DOCTYPE html>
```
Why did you remove `<!DOCTYPE html>`? This is mandatory for HTML5.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+    <script
+      async=""
+      src="https://static.hotjar.com/c/hotjar-3665471.js?sv=6"
+    ></script>
```
Why did you remove the HOTJAR script? Please put it back:

```<!-- Hotjar Tracking Code for https://avocat-nagy-cluj.ro -->
<script>
    (function(h,o,t,j,a,r){
        h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
        h._hjSettings={hjid:3665471,hjsv:6};
        a=o.getElementsByTagName('head')[0];
        r=o.createElement('script');r.async=1;
        r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
        a.appendChild(r);
    })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
</script>```

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+      <nav class="nav">
+        <img
+          src="img/logo.png"
+          alt="Bankist logo"
```
Please add a valid alt attribute for this image. A good one should be "Sebastian Gabriel Nagy | Avocat Cluj logo".

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+          </li>
+        </ul>
+        <button id="toggleButton-close">
+          <svg
```
Please move the .svg files in a separate file and use them with <img src=""> in the HTML so we don't pollute the HTML with tons on useless lines of svg code.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+        </div>
         <!-- <button class="btn--text btn--scroll-to">Mai mult &DownArrow;</button> -->
-        <a class="btn--show-modal" href="#contact">Contactează</a>
+        <button class="btn--show-modal btn--text">Contactează</button>
```
This button should scroll to the bottom of the website, not open the modal. Please put back the `href` attribute.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
-          </svg>
+        <div class="operations__content operations__content--1">
+          <div class="operations__icon operations__icon--1">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 576 512">
```
Please extract the svg in a separate file and use it with the <img src=""> tag in HTML.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
-          </svg>
+        <div class="operations__content operations__content--2">
+          <div class="operations__icon operations__icon--2">
+            <svg
```
Same as above.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+          class="operations__content operations__content--3 operations__content--active"
+        >
+          <div class="operations__icon operations__icon--3">
+            <svg
```
Same as above.

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+        <img
+          src="img/law-1.jpg"
+          data-src="img/law-1.jpg"
+          alt="Computer"
```
Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii".

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+        <img
+          src="img/law-2.jpg"
+          data-src="img/law-2.jpg"
+          alt="Plant"
```
Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii legale".

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+        <img
+          src="img/law-3.jpg"
+          data-src="img/law-3.jpg"
+          alt="Credit card"
```
Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii juridice".

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+        <img
+          src="img/law-1--lazy.jpg"
+          data-src="img/law-1.jpg"
+          alt="Computer"
```
Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii legale si juridice".

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+        <img
+          src="img/law-3--lazy.jpg"
+          data-src="img/law-2.jpg"
+          alt="Plant"
```
Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii legale".

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
+        <img
+          src="img/law-3--lazy.jpg"
+          data-src="img/law-3.jpg"
+          alt="Credit card"
```
Please add meaningful **alt text** to all images. This is very important for accessibility reasons and SEO reasons. Here you can use "Avocat definitiv Cluj-Napoca Sebastian Nagy servicii juridice".

georgebrata commented on index.html:
```diff
@@ -1,534 +1,1010 @@
-          <svg>
-            <use xlink:href="img/icons.svg#icon-upload"></use>
-          </svg>
+      <div class="header__title header__title-m">
```
Do you close the `</header>` element anywhere? It seems like you removed some closing tags and this might introduce bugs, please double-check.

georgebrata reviewed (commented):
Please fix

adonisjoe committed:
fix all highlighted errors

adonisjoe committed:
Update the contact form inside the modal

include action attribute to the form element
